src/skin/tooltips.cpp

```diff
@@ -267,6 +265,19 @@ void Tooltips::addStandardTooltips() {
             << tr("Hold-to-talk or short click for latching to")
             << tr("mix microphone input into the master output.");
 
+    add("talkover_duck_mode")
```

Comment:
Note: These need to be added to all skins.

Reply:
There are also other persistent controls where, we need to make it either accessible or force it to a default after skin change.